onnxscript/function_libs/torch_aten/ops/core.py

```diff
@@ -4208,19 +4208,70 @@ def aten_native_dropout_backward(
     raise NotImplementedError()
 
 
+@torch_op("aten::native_group_norm", trace_only=True)
 def aten_native_group_norm(
-    input: TensorType,
-    weight: Optional[TensorType],
-    bias: Optional[TensorType],
-    N: INT64,
-    C: INT64,
-    HxW: INT64,
-    group: int,
-    eps: float,
-) -> tuple[TensorType, TensorType, TensorType]:
+    input: TFloat,
+    weight: Optional[TFloat],
+    bias: Optional[TFloat],
+    N: Optional[INT64] = None,  # pylint: disable=unused-argument
```

Comment:
For my education. How do we know if an argument can be optional or not?

From aten signature below `native_group_norm(Tensor input, Tensor? weight, Tensor? bias, SymInt N, SymInt C, SymInt HxW, int group, float eps) -> (Tensor, Tensor, Tensor)`

`weight` and `bias` have `?` tagged, hence it's natural. But why are `N`, `C`, etc also optional? (And somewhat related, why does `eps` alone has default value?)

Reply:
Good question. From the signature I don't think they should be optional. I am guessing it is due to the discrepancy in the OpInfo definition? @xiaowuhu 

I am thinking if we should create our opinfo for this op.

Reply:
@justinchuby can explain more here :)
For in ONNX, we don't explicitly need N,C,HxW, because the information are already in the input tensor shape. They cannot have default value.
But, aten_native_group_norm() function must need these values as input.
For 'group' and 'eps', they should be not None. I will change the signature in next commit.

Reply:
If aten signature doesn't specify the default, should we find it's upper level torch function for default? If that's the case, I think eps=1-e5 would be reasonable.